java/libphonenumber/src/com/google/i18n/phonenumbers/nano/Phonemetadata.java

```diff
@@ -2,6 +2,8 @@
 
 package com.google.i18n.phonenumbers.nano;
 
+// @@protoc_insertion_point(imports_scope)
```

Comment:
Yes, the jars need to be regenerated.

This class: *no* idea why there is a change. This happened because I ran mvn clean and mvn build.... nothing to do with my cl, but maybe to do with the protoc version I have on my computer.